src/gui/mainwindow.ui

```diff
@@ -205,7 +205,7 @@
   </action>
   <action name="actionDownload_from_URL">
    <property name="text">
-    <string>Add &amp;link to torrent...</string>
+    <string>Add torrent &amp;link...</string>
```

Comment:
Capitalize: "Add Torrent &amp;Link..."
(same for the rest where this string appears)